DuckDuckGoTests/AutoClearSettingsScreenTests.swift

```diff
@@ -35,7 +35,7 @@ class AutoClearSettingsScreenTests: XCTestCase {
     override func tearDown() {
         super.tearDown()
         
-        AppDependencyProvider.shared = AppDependencyProvider()
+        AppDependencyProvider.shared = AppDependencyProvider.makeTestingInstance()
```

Comment:
I'm purposely not questioning why we're instantiating these in tests as I'd expect tests to simply use a mocked instance.  I think this should go away but also opted not to do that here.

Reply:
It should go away but that's not so urgent here.